snuba/cli/migrations.py

```diff
@@ -128,6 +128,76 @@ def migrate(
     click.echo("Finished running migrations")
 
 
+@migrations.command()
+@click.option("-g", "--group", default=None)
+@click.option(
+    "-r",
+    "--readiness-state",
+    multiple=True,
+    type=click.Choice([r.value for r in ReadinessState], case_sensitive=False),
```

Comment:
To support case insensitivity on [ReadinessState] enum (https://github.com/getsentry/snuba/blob/c636a661b5b3a61aa17e6e63896117572835b7f4/snuba/datasets/readiness_state.py#L6), will we need conversion to upper case?

Reply:
@onkar I believe that the choice options take care of this https://click.palletsprojects.com/en/stable/options/#choice-options